_docs/how_to_run_tests.md

```diff
@@ -121,21 +123,50 @@ export SPEC_FACTS_OS=centos
 export SPEC_FACTS_OS=centos-7
 ```
 
-### Acceptance tests
+### Acceptance tests quickstart
 
 The unit tests just check the code runs, not that it does exactly what we want on a real machine.
 For that we're using [beaker](https://github.com/voxpupuli/beaker).
 
 This fires up a new virtual machine or container and runs a series of simple tests against it after applying the module.
-You can run this on your own with:
+
+You can run beaker on your own with:
 
 ```shell
 BEAKER_SETFILE=centos9-64 bundle exec rake beaker
 ```
 
+**Note**: You will need the `system_tests` gem group, check it has not been excluded when installing requirements (eg. use `bundle config set --local without 'development release'`).
```

Comment:
is the syntax correct? I thought gem groups need to be delimited by `:`, not by whitespace.

Reply:
Yes, line 42 already used whitespaces (bundle config set --local without 'development system_tests release') and it worked, so I guess it's ok?